conf/supervisord.conf

```diff
@@ -17,20 +17,23 @@ stdout_logfile=NONE
 stderr_logfile=NONE
```

Comment:
Just to close the loop on this one:

There's doesn't appear to be a problem with NGINX and missing services. For example if NGINX is configured to connect to the web process on a UNIX file socket and the web process is not running this doesn't seem to cause an issue: NGINX seems to work fine.

I dug up the WebSocket NGINX tuning work that was done previously, here's a [Slack post that I think covers what was done](https://hypothes-is.slack.com/archives/C4K6M7P5E/p1667314619421209?thread_ts=1667214140.419019&cid=C4K6M7P5E) (TLDR: the `worker_rlimit_nofile` and `worker_connections` settings were increased). I think it probably would have been fine to move the WebSocket behind the internal NGINX. The internal NGINX's lower `worker_rlimit_nofile` and `worker_connections` settings might have caused a problem but we could have just increased those in the internal `nginx.conf`. (For the main web process there are lower values for these settings in its external `nginx.conf`, so those values would have remained effective for the web process.)

However, I nonetheless ended up changing this PR so that the WebSocket process will remain *not* behind the internal NGINX in the US WebSocket environments, but *will be* behind the internal NGINX in the Canadian environment. This is the same setup as currently.

This makes things more complicated in a few ways:

* It means we have to have two different WebSocket config files instead of one
* And we need a `WEBSOCKET_CONFIG` envvar for toggling between the config files
* It means that Canada differs from the US in running the WebSocket behind NGINX
* We need to keep the [WebSocket support in `nginx.conf`](https://github.com/hypothesis/h/blob/f8432e53197e9eff5150f79b070e50c01720e9f4/conf/nginx.conf#L57-L67) even though it's only used in the Canada environment.

To that end I'm going to open a follow-up issue (not written yet) to change the Canada environment to  be the same as the US environment in not running the WebSocket behind NGINX. Then we'll be able to remove the above complications from this repo and we should have a clean and simple setup and consistency between US and CA.